<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SYAM-DS0F3D813" name="Test1" tests="1" failures="1" timestamp="8 Jun 2018 10:03:12 GMT" time="11.922" errors="0">
  <testcase name="testMercuryTours" time="6.834" classname="tests.TestApp">
    <failure type="java.lang.AssertionError" message="Intentionally failed expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Intentionally failed expected [false] but found [true]
	at pages.MercuryToursRegisterPage.clickSignOff(MercuryToursRegisterPage.java:30)
	at tests.TestApp.testMercuryTours(TestApp.java:47)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testMercuryTours -->
</testsuite> <!-- Test1 -->
